### DISCUSSION TITLE
How to modify item properties!

### DISCUSSION THREAD

**Discussion by LFTEC** [azerothcore/azerothcore-wotlk#15318] [General]
Hi everyone. Can you teach me how to modify item properties?
While I modify property in acore_world.item_template and restart worldserver, the item don't have any change.
When I copy item to a new one and use .additem, everything gets correct. But when change property then, It will not reflect to worldserver.

**Comment by BrucePrintscreen** [comment]
Hello there,

There is a Tool for chaning item properties and more.
Have a look at Kiera 3, which is the easiest way

https://github.com/azerothcore/Keira3
